### PR TITLE
Pankajktech patch 1

### PR DESCRIPTION
**Pull request by pankajktech**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4127"></a>

### CODE REVIEW

**Review by devvsakib — DISMISSED** [review]
Looks good to merge

**Comment by pankajktech** [comment]
data:pankajktech

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please note that your file should be in data directory. 🛠

You need to rename your file [here](https://github.com/pankajktech/LinkFree/edit/pankajktech-patch-1/pankajktech.json) as `data/pankajktech.json` to move it into the data directory. Let us know if you need any help with this! 
image

File: data/pankajktech.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      "icon": "FaLinkedin"
+    }
+    {
+      "name": "Follow me on Github",
+      "url": "https://www.github.com/pankajktech",
+      "icon": "FaGitHub"
+    },
+  ],
+
+  "Special Milestones"[
```
Comment (kumarsonsoff3) [inline data/pankajktech.json:33]: Invalid milestone key is used! Please update it as suggested. 🛠

```suggestion
  "milestones": [
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/pankajktech.json (hunk 1 above)
Comment (pankajktech) [reply data/pankajktech.json]: Updated

File: data/pankajktech.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    {
+      "name": "Linkedin",
+      "url": "https://www.linkedin.com/in/pankajktech/",
+      "icon": "FaLinkedin"
+    }
+    {
+      "name": "Follow me on Github",
+      "url": "https://www.github.com/pankajktech",
+      "icon": "FaGitHub"
+    },
```
Comment (kumarsonsoff3) [inline data/pankajktech.json:30]: Please remove the extra comma `,` here! 
```suggestion
    }
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Last change needed, then it should be good to go!

File: data/pankajktech.json (hunk 2 above)
Comment (pankajktech) [reply data/pankajktech.json]: Done

File: data/pankajktech.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/pankajthakuroff/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "Linkedin",
+      "url": "https://www.linkedin.com/in/pankajktech/",
+      "icon": "FaLinkedin"
+    }
```
Comment (kumarsonsoff3) [inline data/pankajktech.json:25]: There is another missing comma here. 
```suggestion
    },
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/pankajktech.json (hunk 3 above)
Comment (pankajktech) [reply data/pankajktech.json]: Done

**Review by kumarsonsoff3 — APPROVED** [review]
Finally we are good to go. Thanks for your Contribution! 🎉 

<p align="center"></p>

**Pull request merged**